packages/office-addin-debugging/src/start.ts

```diff
@@ -100,6 +109,16 @@ export async function runDevServer(commandLine: string, port?: number): Promise<
                 } else {
                     throw new Error(`The dev server is not running on port ${port}.`);
                 }
+
+                const manifestInfo: any = await readManifestFile(manifestPath);
+                const sourceLocation: string = manifestInfo.defaultSettings.sourceLocation;
+                const isBuildComplete: boolean = await waitUntilDevBuildIsComplete(sourceLocation);
```

Comment:
Per my previous comment, you can update naming so it is `const isAvailable: boolean = await waitUntilUrlIsAvailable(sourceLocation);`